eloquent/distribution.yaml

```diff
@@ -1699,7 +1702,6 @@ repositories:
       url: https://github.com/ros2-gbp/ros1_bridge-release.git
       version: 0.8.1-4
     source:
-      test_pull_requests: true
```

Comment:
Dashing has `test_commits: false`?

Reply:
I set this because it was passing, but it's not actually doing anything due to lack of ros1.

Removed.